src/bsp/src/stm32/include/UWBMessages.h

```diff
@@ -35,16 +35,16 @@ namespace UWBMessages {
     };
 
     struct TWRPoll {
-        DWFrame m_header_frame;
+        DWFrame m_headerFrame;
     };
 
     struct TWRResponse {
-        DWFrame m_header_frame;
+        DWFrame m_headerFrame;
         uint64_t m_calculatedTOF;
     };
 
-    struct TWRFinal {
-        DWFrame m_header_frame;
+    struct TWRFinal : DWFrame {
```

Comment:
not sure you wan't to inherit here. This will double the size of the message...

Reply:
oops no not supposed to be there